tests/loggen/loggen.c

```diff
@@ -103,6 +103,9 @@ generate_message(char *buffer, int buffer_size, int thread_id, unsigned long seq
   else
     str_len = generate_log_line(buffer, buffer_size, syslog_proto, thread_id, seq);
 
+  if (str_len < 0)
+    return str_len;
```

Comment:
Maybe an explicit `return -1;` would be better.

Reply:
Done.